machine-learning/04-medulloblastoma_LV_differences.Rmd

```diff
@@ -157,13 +161,13 @@ For illustrative purposes, we'll pick an LV and go through using `ggplot2` to
 make a plot.
 
 ```{r}
-lv <- "94,IRIS_Monocyte-Day7"
+lv <- "13,SVM T cells CD4 memory activated"
```

Comment:
I'm guessing it's fine since you put it this way, but before you put `_` instead of spaces, why the change?

Reply:
And should it be `memory-activated`?

Reply:
Should it be capitalized?

Reply:
This is automatically generated by `PLIER`. I need to match the pattern set by PLIER. I don't use this anywhere else (e.g., for setting plot titles).